CONTRIBUTING.md

```diff
@@ -185,6 +185,18 @@ pytest test/test_nn.py -k Loss -v
 The above is an example of testing a change to Loss functions: this command runs tests such as
 `TestNN.test_BCELoss` and `TestNN.test_MSELoss` and can be useful to save keystrokes.
 
+### Reproducing Linux CI errors locally
+1. If CUDA is needed, run `export USE_CUDA_DOCKER_RUNTIME=1`.
+2. Run `source .circleci/scripts/setup_ci_dependencies.sh` to set up local environment.
```

Comment:
Telling users to run this is incredibly questionable. This script unconditionally installs our specific NVIDIA driver and Docker version and will almost assuredly brick any user who tries to run these instructions if they are not specifically on the CircleCI AMI. We should assume that the user has a working nvidia-docker setup (pointing them to install instructions if they do not) and not ask them to run scripts like this.